src/main/java/edu/cornell/kfs/fp/service/AmazonWebServicesBillingService.java

```diff
@@ -0,0 +1,10 @@
+package edu.cornell.kfs.fp.service;
+
+public interface AmazonWebServicesBillingService {
+    
+    /**
+     * This function will call the AWS Billing web service and generate Distribution Of Income Documents.
+     */
+    void generateDistributionOfIncomeDocumentsFromAWSService();
```

Comment:
I edited my javadoc for clarity and to provide some more information.  Bryan has recommended in the past that we document Interfaces, so I think something should be here.